models/base.py

```diff
@@ -13,7 +15,12 @@ def __init__(self, cv=5, n_iter=20):
 
     def fit(self, X, y):
         """Train on a training set and select optimal hyperparameters."""
-        # TODO: Impute missing predictors if necessary and set imputer here.
```

Comment:
**Problem**
NaN values in y to be imputed as well

**Fix**
@jirvin16

Reply:
At this point, there should be no NaN values in `y`. All of the rows with NaN y values (I.e. all of the rows with CH4 we're trying to gapfill) should be filtered out by this point:
https://github.com/stanfordmlgroup/methane-gapfill-ml/blob/master/preprocess/preprocess.py#L60-L61
https://github.com/stanfordmlgroup/methane-gapfill-ml/blob/master/preprocess/preprocess.py#L83-L99
https://github.com/stanfordmlgroup/methane-gapfill-ml/blob/master/preprocess/preprocess.py#L105-L119
Are you seeing an error with NaN values in `y`?